doc/cookbook_vs.md

```diff
@@ -378,7 +378,7 @@ Note that being thread-safe is *not* the same thing as being free-threaded, whic
 ### Short answer
 
 * Use `ConfigureAwait(false)` when writing app-independent library code. Such a library should avoid frequent use of `Task.Wait()` and `Task.Result`.
-* Use `ConfigureAwait(true)` when writing code where a `JoinableTaskFactory` is available.
+* Use `ConfigureAwait(true)` when writing code where a `JoinableTaskFactory` is available. Use `await TaskScheduler.Default;` before CPU intensive tasks.
 
 **Note:** Use of `.ConfigureAwait(true)` is equivalent to awaiting a `Task` directly. Using this suffix is a way to suppress the warning emitted by some analyzers that like to see `.ConfigureAwait(false)` everywhere. Where no such analyzer is active, omitting the suffix is recommended for improved code readability.
 
```

Comment:
Maybe explaining the behavior of those two can help?  I saw it quite often that developers just blindly followed some patterns or guidelines to just get the code to work, and only then ran into problems.

Also I think it need recommend the code to follow a consistent style (unless it has reason not to in some places).  A product can get the worse of the two, if ConfiguredAwait(true) and ConfiguredAwait(false) are mixed in one async call chain?

Reply:
Good point. I've added docs for those points.